client/proc.go

```diff
@@ -32,6 +32,7 @@ type Cmd struct {
 	// If Scrub is set, the process element will automatically be removed from its anchor
 	// when the process exits.
 	Scrub bool
+	Name string
```

Comment:
You should not be including a Name field in the command config. The names of commands are their anchor paths. (Configs and names are thus decoupled.) 

If you simply want to run one-off commands and don't care what they are named, place all one off commands in
anchors like /X.../runproc/RANDOM_ID

If you want the user to specify the anchor of a one-off command that is executed, say, on all machines*, then you can take a name argument somewhere (command-line or otherwise), but you cannot add that name to the Cmd struct.
By the way, the name is really an anchor path without the first part. 

I am not really sure why ever the user would want to name a one-off job that gets executed everywhere.